### ISSUE TITLE
README screenshot....too fast?

### ISSUE THREAD

**Issue by sparty02** [sindresorhus/fkill-cli#12]
This is just an observation/my opinion, so take it with a grain of salt.  I love the idea of fkill, great work!

When I was first looking at the README, the animated demo caught my eye quick and was a nice, concise demo of what fkill is supposed to do.  However, the animation seemed wayyyy to fast and it was a bit hard to follow exactly what was going on.  While I could eventually piece it together, I wonder if a more "real time" speed to the gif would be more "accessible" to someone viewing it.

I think the primary thing that feels to fast is the secondary action of typing in the "auto-complete" (after fkill fires up and shows results).  Maybe just delay for a second at startup to show the list of running processes, then type each character in the autocomplete slower?

I dunno, just some (hopefully) constructive feedback.

**Comment by sindresorhus** [comment]
You're absolutely right. Multiple other people commented it too.
